ios/FluentUI/Tooltip/TooltipView.swift

```diff
@@ -98,12 +86,35 @@ class TooltipView: UIView {
         messageLabel.text = message
         messageLabel.textAlignment = textAlignment
         messageLabel.isAccessibilityElement = false
-        addSubview(messageLabel)
+
+        if let titleLabel = titleLabel, let title = title {
```

Comment:
curious why we need to check both existence of title AND titlelabel. shouldn't title be good enough?

Reply:
we'd have to force unwrap titleLabel otherwise